.github/workflows/easy-cp.yml

```diff
@@ -37,6 +36,7 @@ jobs:
           token: ${{ github.token }}
           path: flutter
           ref: master
+          # Use depth 0 to checkout history of master branch, so the cp commit is a known object
```

Comment:
Yeah so `fetch-depth` means the number of commits to fetch, and `fetch-depth: x` means fetching the last x commits on branch `ref`.

For the cherry pick commit, its unclear where the commit is on the master branch. e.g. the cp commit could be located at 201 commits before HEAD, and therefore `fetch-depth: 200` would not retrieve the cp commit.

I had to do a `fetch-depth:0` here (0 fetches all commits on `ref`), so that no matter where the cp commit is in the commit history, it is always retrieved and a known object. Even if it is the first commit of the repository. 

By default, fetch-depth is set to 1 which only retrieves the tip commit on branch `ref`.

Reply:
Updated the comment to `Checkout all history commits on master branch, so that the cp commit is a known object`

I might have misunderstood and I am good with whichever comment that would make this part clearer. If Casey proposes a change here I d be happy to accept it.